lib/chef/resource/windows_pagefile.rb

```diff
@@ -39,16 +39,26 @@ class WindowsPagefile < Chef::Resource
 
       ```ruby
       windows_pagefile 'Delete the pagefile' do
-        path 'C:\pagefile.sys'
+        path 'C'
         action :delete
       end
       ```
 
+      **Switch to system managed pagefiles**:
+
+      ```ruby
+      windows_pagefile 'Change the pagefile to System Managed' do
+        path 'E:\'
+        system_managed true
+        action :set
+      end
+      ```
+
       **Create a pagefile with an initial and maximum size**:
 
       ```ruby
-      windows_pagefile 'create the pagefile' do
-        path 'C:\pagefile.sys'
+      windows_pagefile 'create the pagefile with these sizes' do
+        path 'f:\'
```

Comment:
do you want to accept only `path 'f:\'` and require the trailing slash or could that be optional and `path 'f:'` would be fine?

Reply:
I can accept both. I split at the colon and verify that way.

Reply:
Per Tim, he wanted users to be able to enter either C, F:\ or G:\\foo\pagefile.sys